feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -807,7 +811,7 @@ func incrementMAC(mac string, i int) (string, error) {
 
 func createFlows(t *testing.T, dut *ondatra.DUTDevice, top gosnappi.Config, aggIDs []string) []gosnappi.Flow {
 	for _, aggID := range aggIDs {
-		dutAggMac = append(dutAggMac, gnmi.Get(t, dut, gnmi.OC().Lacp().Interface(aggID).SystemIdMac().State()))
+		dutAggMac = append(dutAggMac, gnmi.Get(t, dut, gnmi.OC().Interface(aggID).Ethernet().MacAddress().State()))
```

Comment:
Looks like this oc path gnmi.Get(t, dut, gnmi.OC().Interface(aggID).Ethernet().MacAddress().State()) doesn't worked

Reply:
Alternatively , we can use system MAC ( my station mac ) config with static MAC address and use the same for traffic flow creation.  @rahulsy26  Please confirm if this approach work.

Reply:
@bkreddy143 and @LimeHat, Instead of using system MAC we can learn arp on ixia interface and use the same arp in our flow. I tried and it worked to me

Reply:
I am also working on this test to add additional test

Reply:
@rahulsy26 , i don't see aggID struct has aggID.ateAggName and aggID.dutIPv4 unless you change 
for _aggID := range aggId  { }  to 
for _aggID := range []*aggPortData{agg1,agg2,agg3}  {}

Reply:
> FYI I am also working on this test to add additional test

@rahulsy26  if you are adding new tests , please add gRIBI backup NHG functionality test , current test doesn't actual cover backup NHG switchover test which was the actual ask .

Reply:
@bkreddy143 i am working on add gRIBI backup NHG functionality test will update once done

Reply:
updated test to get MAC address from ARP